<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iCalendar export configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="filePathLabel">
     <property name="text">
      <string>Export birthday list in iCalendar file:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filePathLayout">
     <item>
      <widget class="QLineEdit" name="filePathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="filePathButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="alarmsCheckBox">
     <property name="text">
      <string>Set alarms</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="alarmsDelayLayout">
     <item>
      <widget class="QSpinBox" name="alarmsDelaySpinBox">
       <property name="suffix">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="alarmsDelayLabel">
       <property name="text">
        <string>days before each birthday</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="customVeventLabel">
     <property name="text">
      <string>Custom ICS properties for VEVENT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="customVeventTextEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="customValarmLabel">
     <property name="text">
      <string>Custom ICS properties for VALARM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="customValarmTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
